dao/tests/features/voting/repo/repo_voter_actions.feature

```diff
@@ -31,3 +31,14 @@ Scenario: RepoVoter sets the of PostJobDOSFee with activation time
   # voting lasts 12 days
   When 3 days passed
   Then value of PostJobDOSFee is 12345
+
+# Following scenario fails while creating first voting, which is correct behaviour
```

Comment:
Shouldn't this be uncommented?

Reply:
This was the test that replicated the error. Unfortunately, to test the correct behaviour I would have to add a lot of code to the dao world to allow failing with voting creation